core/src/main/java/org/pentaho/platform/util/Emailer.java

```diff
@@ -162,6 +162,10 @@ public String getEmailFromName() {
     return Messages.getInstance().getString( "emailFromName" ); //$NON-NLS-1$
   }
 
+  Properties getProperties() {
```

Comment:
Without the access modifier this method will be visible inside the package but not by subclasses or classes outside the package. Is this what you want?